codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -120,9 +118,55 @@ If there are obsolete binaries in entitlements configuration files, please delet
 update these file paths accordingly.
 ''';
 
+  List<String> engineArtifactFilePaths = [
```

Comment:
I thought the goal with entitlements.txt was to avoid needing this hardcoded list?

Reply:
umm Right yeah it removes the hard coded list of the files within each artifact. my understanding is that the artifact list might still be needed, so we know where to fetch them from/ upload to with respect to the gcloud buckets. (like we wouldn't need to know about darwin-x64/artifacts.zip/FlutterMacOs/framework/blah/blah which is a binary, but we still need the information on the artifact darwin-x64/artifacts.zip so we could fetch it from google cloud)

Reply:
This is something that should be in the v2 engine recipes for each codesign sub build.

The codesign script should generalize the `switch` option to be an `path` option. Then callers will be able to codesign any generic artifact without needing to update this script.

Reply:
Sure, I removed the list from codesign app. Will remember to add them in the engine recipe.